### PR TITLE
nodoc raw_write_attribute

### PR DESCRIPTION
**Pull request by mcfiredrill**
Is this supposed to be public API? If so, I can document it instead.
:memo:

### CODE REVIEW

File: activerecord/lib/active_record/attribute_methods/write.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ def write_attribute(attr_name, value)
         write_attribute_with_type_cast(attr_name, value, true)
       end
 
-      def raw_write_attribute(attr_name, value)
+      def raw_write_attribute(attr_name, value) #:nodoc:
```
Comment (akshay-vishnoi) [inline activerecord/lib/active_record/attribute_methods/write.rb]: add a space after `#`. :smiley:

**Comment by sgrif** [comment]
Seems fine. Space is needed

**Comment by mcfiredrill** [comment]
@sgrif Thanks, space added.

**Comment by arthurnn** [comment]
thanks

**Pull request merged**